### PR TITLE
Forces everyone's badonkers out in the open if they aren't covered + new clothing

### PR DESCRIPTION
**Pull request by KathrinBailey**
## About The Pull Request
• Makes it so wearing clothing that doesn't cover your chest doesn't hide your badonker (technical term) sprite accessory. Full credit for this goes to Azarak.
•  Adds a fashionable skirt to the game.
• Adds a generic military uniform. A variant of the seccie jumpsuit sporting the same stats.
• Adds a plain-looking shirt. You can now have prisoners looking like colonial-era slaves. Features a fun tracking implant like prisoner jumpsuits.

## Why It's Good For The Game
Fixes buggy behaviour with sprite accessories that are always covered up by jumpsuit and suit slot clothing regardless of these items actually "cover up" your bits.
Also, more fashion.

## Images
image
image

## Changelog
:cl:
fix: Wearing clothing that leaves your chest bare now forcefully exposes your badonkers (technical code term).
/:cl:

### CODE REVIEW

**Comment by Nerev4r** [comment]
All of these in the loadout?

**Comment by KathrinBailey** [comment]
> All of these in the loadout?

nah not yet, still wondering what to do with them.

**Comment by KathrinBailey** [comment]
I made the skirt a donator item
image

File: tgstation.dme [hunk 1]
```diff
@@ -3750,7 +3750,6 @@
 #include "modular_skyrat\modules\bluespace_locker\code\controllers\subsystem\bluespace_locker.dm"
 #include "modular_skyrat\modules\bluespace_locker\code\game\area\Space_Station_13_areas.dm"
 #include "modular_skyrat\modules\bluespace_locker\code\game\objects\structures\crates_lockers\bluespace_locker.dm"
-#include "modular_skyrat\modules\bongs\items\bong.dm"
```
Comment (KasparoVy) [inline tgstation.dme:3753]: Bong removed?

File: tgstation.dme [hunk 2]
```diff
@@ -3810,8 +3809,6 @@
 #include "modular_skyrat\modules\customization\modules\client\loadout\uniform.dm"
 #include "modular_skyrat\modules\customization\modules\client\loadout\donator\_donator.dm"
 #include "modular_skyrat\modules\customization\modules\client\loadout\donator\backpack.dm"
-#include "modular_skyrat\modules\customization\modules\client\loadout\donator\feet.dm"
```
Comment (KasparoVy) [inline tgstation.dme:3813]: Ruh roh.

**Comment by KathrinBailey** [comment]
reEeE

**Pull request merged**